src/elm/Page/Community/Settings/Features.elm

```diff
@@ -0,0 +1,259 @@
+module Page.Community.Settings.Features exposing (Model, Msg, init, msgToString, update, view)
+
+import Api.Graphql
+import Community
+import Eos exposing (Symbol)
+import Eos.Account
+import Graphql.Http
+import Html exposing (Html, div, input, label, span, text)
+import Html.Attributes exposing (checked, class, for, id, name, style, type_)
+import Html.Events exposing (onCheck)
+import I18Next exposing (Translations, t)
+import Json.Encode
+import Page
+import Ports
+import Route
+import Session.LoggedIn as LoggedIn exposing (External(..), FeedbackStatus(..))
+import UpdateResult as UR
+
+
+init : LoggedIn.Model -> Symbol -> ( Model, Cmd Msg )
+init { shared } symbol =
+    ( initModel symbol
+    , Api.Graphql.query shared (Community.communityQuery symbol) CompletedLoad
+    )
+
+
+initModel : Symbol -> Model
+initModel symbol =
+    { status = Loading
+    , symbol = symbol
+    , hasShop = False
+    , hasObjectives = False
+    }
+
+
+type alias Model =
+    { status : Status
+    , symbol : Symbol
+    , hasShop : Bool
+    , hasObjectives : Bool
+    }
+
+
+type Status
+    = Loading
+    | LoadingFailed (Graphql.Http.Error (Maybe Community.Model))
+    | Loaded Community.Model
+
+
+type Feature
+    = Shop
+    | Objectives
+
+
+type Msg
+    = CompletedLoad (Result (Graphql.Http.Error (Maybe Community.Model)) (Maybe Community.Model))
+    | ToggleShop Bool
+    | ToggleActions Bool
+    | Ignored
+
+
+type alias UpdateResult =
+    UR.UpdateResult Model Msg (External Msg)
+
+
+view : LoggedIn.Model -> Model -> { title : String, content : Html Msg }
+view loggedIn model =
+    let
+        title =
+            "Features"
+
+        content =
+            let
+                translations =
+                    loggedIn.shared.translations
+
+                translate =
+                    t translations
+            in
```

Comment:
when using the screen I've noticed that the toggles open up as disabled, and suddenly change. We should only show the screen once we have everything we need